packages/astro/src/@types/astro.ts

```diff
@@ -791,6 +791,24 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		assetsPrefix?: string;
+		/**
+		 * @docs
+		 * @name build.assetsRemoveOriginals
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 3.0.6
+		 * @description
+		 * Specifies necessity for removing assets originals.
```

Comment:
The documentation here is quite important, because this option is unsafe. 
```suggestion
		 * Specifies if original files for optimized assets should be removed from the final build in SSG. Note that this option is **potentially unsafe**, as we cannot reliably tell if your website uses the original files outside of the optimization pipeline. 
		 * With this option enabled, **ALL** original files won't be included in the final output.
```